gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/Project.java

```diff
@@ -198,6 +201,71 @@ public Project replace() {
     return resourceManager.replace(this);
   }
 
+  /**
+   * Returns the IAM access control policy for the specified project. Returns {@code null} if the
+   * resource does not exist or if you do not have adequate permission to view the project or get
+   * the policy.
+   *
+   * @throws ResourceManagerException upon failure
+   * @see <a href=
+   *     "https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/getIamPolicy">
+   *     Resource Manager getIamPolicy</a>
+   */
+  public Policy getPolicy() {
+    return resourceManager.getPolicy(projectId());
+  }
+
+  /**
+   * Sets the IAM access control policy for the specified project. Replaces any existing policy.
+   * It is recommended that you use the read-modify-write pattern. See code samples and important
+   * details of replacing policies in the documentation for {@link ResourceManager#replacePolicy}.
+   *
+   * @throws ResourceManagerException upon failure
+   * @see ResourceManager#replacePolicy
```

Comment:
Any reason for this reference (I mean, is necessary and if so why not in others).